garnish/system.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2019-2019 the garnish authors. See copying.md for legal info.
+
+import networkx as nx
+import numpy as np
+from pymol import cmd
+
+
+class System:
+    def __init__(self, sys_dict):
```

Comment:
I like how you can do the the modifications like the elastics fix in the system initialization, nice :D.
We could also put the final structure in the docstring.

Reply:
Oh and also if we have this class, why not initialize it in the first place like this instead of making the dict first.. Seems unnecessary..
Maybe for a next iteration.

Reply:
You mean, refactor the parsing functions into methods?